mqt/qecc/bindings.cpp

```diff
@@ -24,7 +35,71 @@ std::vector<bool> sampleIidPauliErr(const std::size_t length, const double physi
     return Utils::sampleErrorIidPauliNoise(length, physicalErrRate);
 }
 
+py::dict applyEcc(const py::object& circ, const std::string& eccName, const size_t eccFrequency) {
+    auto qc = std::make_shared<qc::QuantumComputation>();
+
+    if (py::isinstance<py::str>(circ)) {
+        auto&& file = circ.cast<std::string>();
+        qc->import(file);
+    } else {
+        py::object const quantumCircuit       = py::module::import("qiskit").attr("QuantumCircuit");
+        py::object const pyQasmQobjExperiment = py::module::import("qiskit.qobj").attr("QasmQobjExperiment");
+        if (py::isinstance(circ, quantumCircuit)) {
+            qc::qiskit::QuantumCircuit::import(*qc, circ);
+        } else if (py::isinstance(circ, pyQasmQobjExperiment)) {
+            qc::qiskit::QasmQobjExperiment::import(*qc, circ);
+        }
+    }
+
+    auto test = std::make_unique<ecc::Id>(qc, eccFrequency);
```

Comment:
This seems like a leftover from playing around.

Reply:
Indeed. I removed the lines.